### PR TITLE
update config and logging for formplayer sentry upgrade

### PR DESCRIPTION
**Pull request by snopoke**
Changes to formplayer config for sentry upgrade.

This needs to be rolled out in conjunction with the formplayer changes: https://github.com/dimagi/formplayer/pull/886
The logging config changes particularly will error if the old version of formplayer is running.

Is it worth trying to split this up into two:
1. Deploy config changes
2. Deploy formpalyer
3. Deploy logging changes (with restart)

##### ENVIRONMENTS AFFECTED
all

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml [hunk 1]
```diff
@@ -120,7 +120,6 @@
     mode: 0644
   with_items:
     - filename: application.properties
-    - filename: sentry.properties
```
Comment (dannyroberts) [inline src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml:123]: Looks like the idea is that by just _not copying_ `sentry.properties` to the new release, we're essentially "deleting" it, and that's why there's no cleanup task necessary here. Took me a second, but makes sense

**Review by dannyroberts — APPROVED** [review]
Is it going to be a challenge at all to coordinate this with the corresponding formplayer release at all? I imagine if we time it right we just expect third parties to always deploy the latest formplayer from the latest commcare cloud and it should just work unless someone tries to do something outside of that model

File: src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml (hunk 1 above)
Comment (snopoke) [reply src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml:123]: Yes, that's correct.

**Comment by snopoke** [comment]
> Is it going to be a challenge at all to coordinate this with the corresponding formplayer release at all? I imagine if we time it right we just expect third parties to always deploy the latest formplayer from the latest commcare cloud and it should just work unless someone tries to do something outside of that model

Thanks for reminding me of this. I had forgotten that we deploy the config along with the each release.

**Pull request merged**